src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -51,6 +51,7 @@ function tryValidate(typed: backend.Backend) {
       environmentVariables: "omit",
       uri: "omit",
       sourceUploadUrl: "omit",
+      invoker: "object",
```

Comment:
Array. We'll have to make the container contract return an array regardless of whether the dev passed a scalar.

Reply:
Changed